e2e/tests/query.feature

```diff
@@ -0,0 +1,341 @@
+Feature: Pocket Query Commands
+
+  Scenario: To show existing commands within the pocket query section
+	Given the user has a pocket client
+	When the user runs the command "query"
+	Then the user should be able to see standard output containing "Usage:"
+
+  Scenario: To query an existing account
+	Given the user is running the network "single_node_network"
+	Given the user has a pocket client
+	When the user runs the command "query account {{address}}" against validator 0
+	Then the user should be able to see standard output containing "/v1/query/account"
+	And the user should be able to see standard output containing "address"
+
+  Scenario: To query an existing account, wrong address
+	Given the user is running the network "single_node_network"
```

Comment:
I recommend the use of hooks here to take advantage of reusability. Lots of repeated steps. Background would work for this case.

Reply:
I agree that some scenarios could have fewer visible steps by using Background; as mentioned in the PR's initial comment, though, we're looking for accuracy in the gherkin side, so I think this can stay as is.